cocos2d/core/CCCamera.js

```diff
@@ -1,282 +0,0 @@
- furnished to do so, subject to the following conditions:
-
- The above copyright notice and this permission notice shall be included in
- all copies or substantial portions of the Software.
-
- THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
- IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
- FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
- AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
- LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
- OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
- THE SOFTWARE.
- ****************************************************************************/
-
-/**
- * <p>
- *     A CCCamera is used in every CCNode.                                                                                 <br/>
- *     The OpenGL gluLookAt() function is used to locate the camera.                                                       <br/>
- *                                                                                                                         <br/>
- *     If the object is transformed by any of the scale, rotation or position attributes, then they will override the camera.          <br/>
- *                                                                                                                                     <br/>
- *     IMPORTANT: Either your use the camera or the rotation/scale/position properties. You can't use both.                            <br/>
- *     World coordinates won't work if you use the camera.                                                                             <br/>
- *                                                                                                                                     <br/>
- *     Limitations:                                                                                                                    <br/>
- *     - Some nodes, like CCParallaxNode, CCParticle uses world node coordinates, and they won't work properly if you move them (or any of their ancestors)           <br/>
- *     using the camera.                                                                                                               <br/>
- *                                                                                                                                     <br/>
- *     - It doesn't work on batched nodes like CCSprite objects when they are parented to a CCSpriteBatchNode object.                  <br/>
- *                                                                                                                                     <br/>
- *     - It is recommended to use it ONLY if you are going to create 3D effects. For 2D effecs, use the action CCFollow or position/scale/rotate. *
- * </p>
- */
-cc.Camera = cc._Class.extend({
```

Comment:
Camera is used by very few feature and have a cost to the renderer, as 3d on web is in our research, new 3d camera will probably come with the new 3d renderer, and it would probably be different with this one. So removing it is a reasonable choice